Plugins/NativeCamera/Source/NativeVideo.cpp

```diff
@@ -149,13 +149,18 @@ namespace Babylon::Plugins
         if (!m_IsPlaying)
         {
             m_IsPlaying = true;
-            NativeCameraImpl->Open(m_width, m_height, m_frontCamera).then(m_runtimeScheduler, arcana::cancellation::none(), [this, env, deferred](const arcana::expected<void, std::exception_ptr>& result) {
+            NativeCameraImpl->Open(m_width, m_height, m_frontCamera).then(m_runtimeScheduler, arcana::cancellation::none(), [this, env, deferred](const arcana::expected<const Camera::CameraDimensions*, std::exception_ptr>& result) {
                 if (result.has_error())
                 {
                     deferred.Reject(Napi::Error::New(env, result.error()).Value());
                 }
                 else
                 {
+                    if (result.has_value()) {
+                        auto cameraDimensions{result.value()};
+                        this->m_width = cameraDimensions->width;
+                        this->m_height = cameraDimensions->height;
```

Comment:
So now we *update* the width/height after the open call returns, but we still initially store the max width/height as the width/height. Do you know how this compares to what happens on the web? I kind of would expect the width/height to be undefined until open or something.

Reply:
I think this effectively does not matter, because we won't attempt to read these properties until after Open is called, but I will separate out maxWidth and width properties and return undefined until we call open (confirmed that matches the browser implementation.)